src/components/accordion/accordionWrapper.js

```diff
@@ -18,8 +22,14 @@ const AccordionWrapper = styled.div`
     border-radius: 4px;
     outline: none;
     background-color: ${props => props.theme.colors.lightGrey} !important;
```

Comment:
you can combine background colour, image, position and repeat in a single rule!

Reply:
Nice spot, changed!